src/ngraph/runtime/gpu/gpu_invoke.hpp

```diff
@@ -25,10 +25,12 @@ namespace ngraph
         namespace gpu
         {
             struct GPURuntimeContext;
-            extern "C" void invoke_primitive(GPURuntimeContext* ctx,
+            extern "C" void invoke_primitive(const GPURuntimeContext* ctx,
```

Comment:
I'm thinking do we really need to pass ctx pinter? will it change dynamically?

Reply:
@fengleitian and I discussed offline, but the summary is that `ctx` is indeed not needed at compile time. However, the reason for establishing the interface so that it requires `ctx` is to support codegen kernels such as https://github.com/NervanaSystems/ngraph/blob/8834039c02b52b88285ed950444421da34d4ecbe/src/ngraph/runtime/gpu/gpu_emitter.cpp#L917, where looking up the primitive needs to run through the `ctx` pointer.